wechaty-puppet/schemas/puppeteventname_string.go

```diff
@@ -12,26 +12,27 @@ func _() {
 	_ = x[PuppetEventNameFriendShipConfirm-1]
 	_ = x[PuppetEventNameFriendShipReceive-2]
 	_ = x[PuppetEventNameFriendShipVerify-3]
-	_ = x[PuppetEventNameLogin-4]
-	_ = x[PuppetEventNameLogout-5]
-	_ = x[PuppetEventNameMessage-6]
-	_ = x[PuppetEventNameRoomInvite-7]
-	_ = x[PuppetEventNameRoomJoin-8]
-	_ = x[PuppetEventNameRoomLeave-9]
-	_ = x[PuppetEventNameRoomTopic-10]
-	_ = x[PuppetEventNameScan-11]
-	_ = x[PuppetEventNameDong-12]
-	_ = x[PuppetEventNameError-13]
-	_ = x[PuppetEventNameHeartbeat-14]
-	_ = x[PuppetEventNameReady-15]
-	_ = x[PuppetEventNameReset-16]
-	_ = x[PuppetEventNameStop-17]
-	_ = x[PuppetEventNameStart-18]
+	_ = x[PuppetEventNameFriendShip-4]
+	_ = x[PuppetEventNameLogin-5]
+	_ = x[PuppetEventNameLogout-6]
+	_ = x[PuppetEventNameMessage-7]
+	_ = x[PuppetEventNameRoomInvite-8]
+	_ = x[PuppetEventNameRoomJoin-9]
+	_ = x[PuppetEventNameRoomLeave-10]
+	_ = x[PuppetEventNameRoomTopic-11]
+	_ = x[PuppetEventNameScan-12]
+	_ = x[PuppetEventNameDong-13]
+	_ = x[PuppetEventNameError-14]
+	_ = x[PuppetEventNameHeartbeat-15]
+	_ = x[PuppetEventNameReady-16]
+	_ = x[PuppetEventNameReset-17]
+	_ = x[PuppetEventNameStop-18]
+	_ = x[PuppetEventNameStart-19]
 }
 
-const _PuppetEventName_name = "PuppetEventNameUnknownPuppetEventNameFriendShipConfirmPuppetEventNameFriendShipReceivePuppetEventNameFriendShipVerifyPuppetEventNameLoginPuppetEventNameLogoutPuppetEventNameMessagePuppetEventNameRoomInvitePuppetEventNameRoomJoinPuppetEventNameRoomLeavePuppetEventNameRoomTopicPuppetEventNameScanPuppetEventNameDongPuppetEventNameErrorPuppetEventNameHeartbeatPuppetEventNameReadyPuppetEventNameResetPuppetEventNameStopPuppetEventNameStart"
+const _PuppetEventName_name = "PuppetEventNameUnknownPuppetEventNameFriendShipConfirmPuppetEventNameFriendShipReceivePuppetEventNameFriendShipVerifyPuppetEventNameFriendShipPuppetEventNameLoginPuppetEventNameLogoutPuppetEventNameMessagePuppetEventNameRoomInvitePuppetEventNameRoomJoinPuppetEventNameRoomLeavePuppetEventNameRoomTopicPuppetEventNameScanPuppetEventNameDongPuppetEventNameErrorPuppetEventNameHeartbeatPuppetEventNameReadyPuppetEventNameResetPuppetEventNameStopPuppetEventNameStart"
```

Comment:
This part is too difficult to read or maintain I believe.

Could we use a more readable style to construct this array?

Reply:
This file does not need to be maintained. It is automatically generated by // go: generate stringer -type = PuppetEventName. see https://godoc.org/golang.org/x/tools/cmd/stringer

before:
```go
type PuppetEventName uint8

const (
  PuppetEventNameUnknown PuppetEventName = iota
  PuppetEventNameLogin
)

fmt.Println(PuppetEventNameLogin) // output: 1
```

after:
```go
//go:generate stringer -type=PuppetEventName
type PuppetEventName uint8

const (
  PuppetEventNameUnknown PuppetEventName = iota
  PuppetEventNameLogin
)

fmt.Println(PuppetEventNameLogin) // output: PuppetEventNameLogin
```

Reply:
Got it, thanks!